### PR TITLE
Plugin: Add update button to plugin list header

### PR DESCRIPTION
**Pull request by enejb**
Before:
screen shot 2016-01-21 at 17 35 29

After:
screen shot 2016-01-21 at 17 35 37

To test:
Visit http://calypso.localhost:3000/plugins 
go into bulk edit mode select a plugin that needs updating click the update button. 

Make sure that everything works as expected when you are on smaller screens. And the are able to update the plugin there as well.

### CODE REVIEW

**Comment by rickybanister** [comment]
It looks right—does it work alright when the options collapse to a dropdown?

File: client/my-sites/plugins/plugin-list-header/index.jsx [hunk 1]
```diff
@@ -44,13 +50,41 @@ export default React.createClass( {
...
+		window.addEventListener( 'resize', this.debouncedAfterResize );
+		_actionBarVisible = findDOMNode( this ).offsetWidth > 719;
+	},
+
+	componentWillUnmount() {
+		window.removeEventListener( 'resize', this.debouncedAfterResize );
+	},
+
+	afterResize() {
+		const actionBarVisible = findDOMNode( this ).offsetWidth > 719
```
Comment (lezama) [inline client/my-sites/plugins/plugin-list-header/index.jsx]: We could extract `719` to a `const`

Comment (johnHackworth) [inline client/my-sites/plugins/plugin-list-header/index.jsx]: ^ I like how this mustachioed guy thinks

**Comment by johnHackworth** [comment]
I think this improves the current experience for those with languages that are too long for our current button size. It would be better if we could manage to calculate the width of the button set and decide if we show it or the dropdown based on that, but this is a good first step

**Comment by enejb** [comment]
@lezama and @johnHackworth What else do you I need to do here before it is ready to merge?

**Comment by johnHackworth** [comment]
@enejb in my opinion it's ready to go... it would be nice to extract that 719 to a MIN_VIEWPORT_WIDTH constant (or similar), but apart from that :+1:

File: client/my-sites/plugins/plugin-list-header/index.jsx [hunk 2]
```diff
@@ -44,13 +53,41 @@ export default React.createClass( {
...
+	afterResize() {
+		const actionBarVisible = findDOMNode( this ).offsetWidth > MIN_VIEWPORT_WIDTH;
+		this.setState( { actionBarVisible } );
+	},
+
 	onBrowserLinkClick() {
 		analytics.ga.recordEvent( 'Plugins', 'Clicked Add New Plugins' );
 	},
 
 	canAddNewPlugins() {
```
Comment (gravityrail) [inline client/my-sites/plugins/plugin-list-header/index.jsx]: This is perhaps more concisely expressed as:

```javascript
return config.isEnabled( 'manage/plugins/browser' ) && !this.props.isWpCom;
```

**Comment by gravityrail** [comment]
lgtm :+1:

**Pull request merged**